src/core/BUILD

```diff
@@ -5194,6 +5236,20 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
+    name = "lru_cache",
+    hdrs = [
+        "util/lru_cache.h",
```

Comment:
We'd like to eventually have one BUILD file per directory, but that will require some internal component changes.  We want to finish the directory reorg first, so that we don't have to do those component changes more than once.

Right now, aside from tests, there are only two BUILD files: the top-level one, and this one in src/core.  We shouldn't create any others until we're ready to start making those internal component changes.